llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -491,6 +491,9 @@ class Triple {
   /// For example, "fooos1.2.3" would return "1.2.3".
   StringRef getEnvironmentVersionString() const;
 
+  /// getSystemCharset - Get the system charset of the triple.
+  StringRef getSystemCharset() const;
+
```

Comment:
Lets call that `DefaultTextEnding" (on most platforms we ignore whatever the system does by default)

Reply:
Thanks, I added Narrow to the name here too